lib/sm/client.rb

```diff
@@ -356,11 +356,44 @@ def post_create_message(args = {})
     def post_create_message_with_attachment(args = {})
       validate_create_context(args)
 
+      Rails.logger.info('MESSAGING: post_create_message_with_attachments')
       custom_headers = token_headers.merge('Content-Type' => 'multipart/form-data')
```

Comment:
Was this log intentionally left in?

Reply:
great question Kyle, yes that is intentional for easier troubleshooting through logs since it is all the same endpoint